src/app/Scenes/SellWithArtsy/SubmitArtwork/SubmitArtwork.tsx

```diff
@@ -136,7 +136,8 @@ export const SubmitSWAArtworkFlow: React.FC<SubmitSWAArtworkFlowProps> = ({
       }
     } catch (error) {
       // Error with mutation.
-      captureMessage(JSON.stringify(error))
+
+      captureMessage(`createOrUpdateSubmission ${JSON.stringify(error)}`)
```

Comment:
very nitpicky 😅 feel free to ignore! 
```suggestion
      captureMessage(`createOrUpdateSubmission ${JSON.stringify(error)}`)
```